packages/react-reconciler/src/__tests__/ReactUpdaters-test.internal.js

```diff
@@ -432,11 +427,7 @@ describe('updaters', () => {
       triggerError();
     });
     expect(Scheduler).toHaveYielded(['onCommitRoot', 'error', 'onCommitRoot']);
-    expect(allSchedulerTypes).toHaveLength(2);
-    expect(allSchedulerTypes[0]).toHaveLength(1);
-    expect(allSchedulerTypes[0]).toContain(Parent);
-    expect(allSchedulerTypes[1]).toHaveLength(1);
-    expect(allSchedulerTypes[1]).toContain(ErrorBoundary);
+    expect(allSchedulerTypes).toEqual([[Parent], [ErrorBoundary]]);
```

Comment:
I didn't spot this before but it seems odd that we only see the topmost component in the updaters when an error is triggered. For all the other tests the the updater is `null` for the initial render.